index.html

```diff
@@ -23,9 +23,114 @@ <h1 class="topic">Okolo <br> Oshoke</h1>
                 Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
                 If you like what you see and have a project you need coded, don’t hesitate and contact me.
             </p>
-            <button class="button1">Start collaboration</button>
+            <button type="button" class="button1">Start collaboration</button>
             <img src="img/image_geometry_2.svg" alt="">
         </section>
+    </div>
+    <section class="gridcontainer works">
+        <h1 class="topic item1">Projects</h1>
+        <div class="item item2">
+            <img src="img/project_1.jpg" alt="laptop">
+            <h1 class="project-name">Project name goes here </h1>
+            <ul class="project-tools">
+                <li class="tool-label">HTML/CSS</li>
+                <li class="tool-label">Ruby on Rails</li>
+                <li class="tool-label">JavaScript</li>
```

Comment:
- Please as requested by the previous reviewer,  for all technology use, try to make it as similar to the Figma design. add some padding to it, and use the padding on the Figma.
##### Figma design 
![Screenshot from 2022-06-01 02-03-14](https://user-images.githubusercontent.com/58771507/171307861-15770fda-6c55-403b-a29d-0a5565246412.png)

##### Your design 

![Screenshot from 2022-06-01 02-02-43](https://user-images.githubusercontent.com/58771507/171307908-58b2c852-1d74-4e60-85a5-0ea3b765c30f.png)